ctags

```diff
@@ -0,0 +1,8 @@
+--regex-ruby=/(^|[:;])[ \t]*([A-Z][[:alnum:]_]+) *=/\2/c,class,constant/
+--regex-ruby=/^[ \t]*attr_(reader|writer|accessor) (:[a-z0-9_]+, )*:([a-z0-9_]+)/\3/A,attr/
+--regex-ruby=/^[ \t]*context ['"](.*)['"].* do/\1/C,rspec context/
```

Comment:
So, the `attr_reader` line lets me `Ctrl-]` from `frump` to `att_reader :frump`, right?

But what does context/describe/feature/it/scenario let me do? Where might I jump from, and where would I land?